jabkit/src/main/java/org/jabref/cli/CheckConsistency.java

```diff
@@ -65,7 +65,9 @@ public void run() {
         List<BibEntry> entries = databaseContext.getDatabase().getEntries();
 
         BibliographyConsistencyCheck consistencyCheck = new BibliographyConsistencyCheck();
-        BibliographyConsistencyCheck.Result result = consistencyCheck.check(entries);
+        BibliographyConsistencyCheck.Result result = consistencyCheck.check(entries, (count, total) -> {
+            System.out.println(Localization.lang("Checking consistency for entry type %0 of %1", count + 1, total));
```

Comment:
Should only happen, if `--porcelain` is NOT active.

Reply:
fixed

Reply:
Forgot to push or is the check somewhere else? 😅

Reply:
forgot to push